app/javascript/pages/getAssistance/privacy.tsx

```diff
@@ -1,12 +1,133 @@
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.infoSharingTitle")}</h2>
+              <ul className="list-disc ml-7">
+                <li>{t("privacyPolicy.infoSharingP1")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.analyticsTitle")}</h2>
+              <p>
+                {renderMarkup(
+                  `${t("privacyPolicy.analyticsP1", {
+                    termsLink:
+                      '<a href="http://www.google.com/analytics/terms/us.html" target="_blank">',
+                    privacyLink:
+                      '<a href="http://www.google.com/policies/privacy/" target="_blank">',
+                    optOutLink:
+                      '<a href="https://tools.google.com/dlpage/gaoptout?hl=en" target="_blank">',
+                    linkEnd: "</a>",
+                  })}`
+                )}
+              </p>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.linksTitle")}</h2>
+              <ul className="list-disc ml-7 space-y-2.5">
+                <li>{t("privacyPolicy.linksP1")}</li>
+                <li>{t("privacyPolicy.linksP2")}</li>
+                <li>{t("privacyPolicy.linksP3")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.siteSecurityTitle")}</h2>
+              <ul className="list-disc ml-7 space-y-2.5">
+                <li>{t("privacyPolicy.siteSecurityP1")}</li>
+                <li>{t("privacyPolicy.siteSecurityP2")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.policyChangesTitle")}</h2>
+              <ul className="list-disc ml-7">
+                <li>{t("privacyPolicy.policyChangesP1")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.questionsTitle")}</h2>
+              <p>
+                {renderMarkup(
+                  `${t("privacyPolicy.questionsP1", {
+                    telLink: '<a href="tel:4157015500">415-701-5500</a>',
```

Comment:
Just wondering if there's a reason we don't use translations for this phone number. Is it because nested `t()` calls are not supported?

Reply:
Sorry, I meant that we have the phone number stored in this json:
https://github.com/SFDigitalServices/sf-dahlia-web/blob/be9da3a21a01b5a5ad3896d1a38a174ad904f0dc/app/assets/json/translations/react/en.json#L2

I think it would be better to use that, rather than hard-code the number here. But maybe it is not possible to do that?

Reply:
Cool, looks like it is possible to nest `t()` calls. I do think that the name of the json key for this phone number could be better, but that is out of the scope of this ticket.